python/tests/test_account.py

```diff
@@ -720,6 +720,7 @@ def test_one_account_send_bcc_setting(self, acfactory, lp):
 
         lp.sec("ac1: setting bcc_self=1")
         ac1.set_config("bcc_self", "1")
+        ac1.set_config("dont_move_to_sentbox", "1")
```

Comment:
Why can't the second device find BCC-self message when it is moved to sentbox? It should idle on sentbox, why doesn't it?

Reply:
The problem is that direct_imap can only IDLE on one folder at a time (currently INBOX). Other solutions would be:
- Somehow use Python threads to IDLE on both folders at once, disadvantage: quite some effort
- check if there is a "Sent" folder and if yes IDLE there, otherwise on Inbox, disadvantage: makes test less deterministic

current solution has the disadvantage that we need a special test config of course.

Reply:
Maybe name this option `SentboxMove` (like `MvboxMove`), and make it user-configurable? Maybe even off by default, as it was the default previously.

Reply:
Hmmm... Why make it user-configurable? I mean, why would anyone want to have all of the sent emails in the inbox?

Reply:
I have BCC-self enabled in my Thunderbird, so I can see both my replies and others messages in the thread.

Anyway, since the problem is that accounts are reused, maybe delete the Sent folder at the end of tests that create it, or prevent account reuse in `SessionLiveConfigFromURL` (this would not help `SessionLiveConfigFromFile` but maybe deprecate it? cc @hpk42 @Simon-Laux)

Reply:
Completely preventing account reuse and deprecating `SessionLiveConfigFromFile` just for this one issue seems overkill?
BTW I listed two other possible solutions above, here comes a third (and I actually already did it for some other tests that were failing for the same reason):
- Create a "Sent" folder at the beginning of the test and then let `direct_imap` watch the sent folder. Disadvantage: quite some effort for every test & makes the test more complex. Very big advantage of doing it at least for some tests: We test that messages are moved to the "Sent" folder (but that's not an argument in favor of doing it here, because these other tests already test this)

Reply:
What about deleting `Sent` folder in `dc_account_extra_configure`? We already delete messages there, deleting all created folders should bring the state of the mailbox closer to the starting point.

Reply:
Ah cool, you just commited it.